### ISSUE TITLE
npm run dev does not work

### ISSUE THREAD

**Issue by kleenkanteen** [supabase-community/chatgpt-your-files#8]
# Bug report

## Describe the bug
When I run npm run dev, open localohst:3000, and try to sign up, after clicking sign up it gives me the error "Error: Hydration failed because the initial UI does not match what was rendered on the server." That was with the default starter.

**Comment by gregnr** [comment]
Hey @kleenkanteen thanks for reporting. Just tried a clean install on my end and not able to reproduce the issue. Would you mind sharing:
- OS
- Browser

Also just confirming you have the latest dependencies installed:
```shell
npm install
```

And you are working off the `main` branch?

Thanks!

**Comment by kleenkanteen** [comment]
Just did a fresh install. Followed the steps to deploy the functions and push migrations to my db. No issues now.

**Comment by gregnr** [comment]
Strange! Glad it's working now though 👍

**Comment by iwo9** [comment]
Hi, I'm having the same issue - could you detail exactly what you did to address it? Did you reinstall node?

**Comment by iwo9** [comment]
> Just did a fresh install. Followed the steps to deploy the functions and push migrations to my db. Can sign up without errors now. Odd. Am on W10, Brave browser, latest versions for both. Node 20.10. On main branch.

Hi, I'm having the same issue - could you detail exactly what you did to address it? Did you reinstall node?

**Comment by iwo9** [comment]
@gregnr could we reopen the issue?

**Comment by kleenkanteen** [comment]
Didn't reinstall node. Did a fresh clone. Try reinstalling node if that
doesn't work.

**Comment by iwo9** [comment]
That's so weird. I've reinstalled node, supabase, and done a fresh clone multiple times each but keep getting the same issue

**Comment by kleenkanteen** [comment]
Try cleaning your npm cache. And if that doesn't work, downgrading the
nextjs version.

I recommend not following this repo btw because the chunking and retrieval
is very poor without a LLM framework. I tried it on a 1000 word file and it
retrieved way too much. Filled up the max tokens I could pass into gpt 3.5.
Try using haystack instead for chunking and retrieval. I did a bunch of
research and it's better designed than langchain and llamaindex.
